content/2022-02-03-be-a-guest-blogger.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Be a Guest Blogger!"
+slug: "2022-02-03-be-a-guest-blogger"
+description: "We want to help you share the interesting things you're doing with Brigade"
+date: "2022-02-03"
+author: "Kent Rancourt"
+summary: "We want to help you share the interesting things you're doing with Brigade."
+---
+
+Hello, Brigadiers!
+
+If you're doing something interesting with Brigade, we want to hear from you --
+and so does the community, so today we wanted to advertise that guest bloggers
+(and vloggers) are welcome!
```

Comment:
Also "vlogger" sounds like an insta influencer who spun up a tiktok channel. I'd say "we want you to contribute your experiences to the Brigade blog(link) and/or Brigade youtube channel(link)."

Reply:
I might have overestimated the ubiquity of that term. Happy to fix this!

Reply:
Fixed in ed4554183f14ca2a968da573b77b5bdada72e949